lib/determine-basal/determine-basal.js

```diff
@@ -175,6 +176,30 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
     // adjust that for deviation like we did eventualBG
     var snoozeBG = naive_snoozeBG + deviation;
 
+    // adjust target BG range if needed to safely bring down high BG faster without causing lows
+    if ( bg > max_bg && profile.adjust_targets_when_high ) {
```

Comment:
isn't it more nicer to let the feature toggle be the first in the if-statement, so `if ( profile.adjust_targets_when_high && bg > max_bg )`